README.md

```diff
@@ -29,7 +29,7 @@ First it will return a reference, a token back to the client that will uniquely
 This token is used throughout the application to tie every aspect of the process to this unique identifier and is later used by the client to retrieve the resources available for this request. 
 The second task for the Palisade Service is to initiate the processing of the request by forwarding it onto the next service in the sequence (in this case the User Service).
 The service will perform its required task and then forward the message onto the next service with the end goal of producing resources ready to be viewed by the client.
-This sequence of steps is completed with the Filtered-Resource Service prepared to provide the processed data.
+This sequence of steps is completed with the Filtered-Resource Service prepared to provide the processed request.
```

Comment:
Late in the day, but may be relevant.  In the documentation for Filtered Resource Service there was also the use of resource in the same context, e.g. "The Filtered-Resource Service sits at the end of the stream pipeline and has the task of returning resources (and their connection details) back to the client. "  This give the idea that there is more than data.

Reply:
Hmm, good spot. Made a small change within the FRS ReadMe file